egs/gale_arabic/s5d/local/process_xml.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+import sys
+import argparse
+
+try:
+    from bs4 import BeautifulSoup
+except ImportError as error:
+    print(sys.argv[0] + ": ImportError,", error)
+    print(sys.argv[0] + ": BeautifulSoup4 is needed to process xml file. You can install it by 'pip install beautifulsoup4'.")
```

Comment:
This is nice, but It would be good to add a new script, local/check_tools.sh that would check for the packages also (and would be run in the beginning of the run.sh. For parsing xml, you will need lxml package also (which is the reason it would be better to add the script and check for it explicitly)

Reply:
OK